README.md

```diff
@@ -73,9 +73,11 @@ For automated tests of the complete example using [bats](https://github.com/bats
 (which tests and deploys the example on AWS), see [test](test).
 
 ```hcl
-module "example" {
+module "inspector" {
   source = "https://github.com/cloudposse/terraform-aws-inspector.git?ref=master"
-  example = "Hello world!"
+  
+  create_iam_role = true
+  enabled_rules   = ["arn:aws:inspector:us-east-2:646659390643:rulespackage/0-m8r61nnh"]
```

Comment:
Please also add `https://docs.aws.amazon.com/inspector/latest/userguide/inspector_rules-arns.html` to the `references:` section

Reply:
6da6e8128fb5cb7aa5607848a050acd35da7d13d